version.txt

```diff
@@ -1 +1 @@
-v5.13.1
+v5.13.2
```

Comment:
```suggestion
v5.14.0
```

it's a minor version bump when adding a new action

Reply:
Done